apps/test/unit/lib/kits/maker/PlaygroundComponentsTest.js

```diff
@@ -0,0 +1,187 @@
+/** @file Playground Component setup tests */
+import five from 'johnny-five';
+import Playground from 'playground-io';
+import {expect} from '../../../../util/configuredChai';
+import sinon from 'sinon';
+import {
+  initializeAccelerometer,
+  initializeButton,
+  initializeColorLeds,
+  initializeLightSensor,
+  initializeSoundSensor,
+  initializeThermometer,
+  initializeTouchPads,
+} from '@cdo/apps/lib/kits/maker/PlaygroundComponents';
+import TouchSensor from '@cdo/apps/lib/kits/maker/TouchSensor';
+
+// Polyfill node's process.hrtime for the browser, gets used by johnny-five.
+process.hrtime = require('browser-process-hrtime');
+
+describe('Circuit Playground Components', () => {
+  let board;
+
+  beforeEach(() => {
+    board = newBoard();
+  });
+
+  describe('initializeColorLeds()', () => {
+    it('initializes all ten LEDs', () => {
+      const components = initializeColorLeds(board);
+
+      // Return ten controllers
+      expect(Object.keys(components)).to.deep.equal([
+        '0', '1', '2', '3', '4', '5', '6', '7', '8', '9'
+      ]);
+
+      // Every returned controller is a five.Led.RGB
+      Object.keys(components).forEach(key => {
+        expect(components[key]).to.be.an.instanceOf(five.Led.RGB);
+      });
+
+      // 20 sysex calls during initialization, two per LED.
+      for (let i = 0; i < 10; i++) {
+        // TODO (bbuchanan): Record what these calls mean.
+        expect(board.io.sysexCommand.getCall(i*2)).to.have.been.calledWith([0x40, 0x10, i, 0, 0, 0, 0]);
+        expect(board.io.sysexCommand.getCall(i*2+1)).to.have.been.calledWith([0x40, 0x11]);
+      }
+    });
+
+    it('can initialize a second set of leds with a second board', () => {
```

Comment:
This test in particular is making sure that the components in question attach to the board we passed in, not just to the first board created, which will be important for reset purposes.